docs/en_US/QuickStart.md

```diff
@@ -157,7 +157,7 @@ Run the **config_windows.yml** file from your command line to start MNIST experi
     nnictl create --config nni/examples/trials/mnist/config_windows.yml
 ```
 
-Note, **nnictl** is a command line tool, which can be used to control experiments, such as start/stop/resume an experiment, start/stop NNIBoard, etc. Click [here](NNICTLDOC.md) for more usage of `nnictl`
+Note, **nnictl** is a command line tool, which can be used to control experiments, such as start/stop/resume an experiment, start/stop NNIBoard, etc. Click [here](Nnictl.md) for more usage of `nnictl`
```

Comment:
"Nnictl" seems a little weird....

Reply:
This is for a unified doc's name.
For **.md** file, we use the uppercase for initial letters, no spacers. For example BuiltinTuners.md, BatchTuner.md.

Reply:
I understand. But can we have an exception for "NNI"? How about "NNIctl.md"?

Reply:
We "Treat all abbreviate words as one word. For example, PaiMode.md, SquadEvolutionExamples.md", have an exception for "NNI" maybe not a standard use?

Reply:
I think either one is okay. Since this is only file name, users would not care too much about it, let's simply unify the format, i.e., using 'Nni'?